index.js

```diff
@@ -402,10 +402,10 @@ class Hyperbee extends ReadyResource {
     return new Watcher(this, range, opts)
   }
 
-  async getAndWatch (key) {
+  async getAndWatch (key, opts) {
     if (!this._watchers) throw new Error('Can only watch the main bee instance')
 
-    const watcher = new EntryWatcher(this, key)
+    const watcher = new EntryWatcher(this, key, opts)
```

Comment:
Should prob default to `this.keyEncoding` and `this.valueEncoding` if not specified

Reply:
Indirectly it already does--the tests check this, so if it ever changes those tests will fail (see test `getAndWatch uses the default encodings of the bee`)

Reply:
Ah gotcha :+1: